addons/mail_bot/static/src/messaging/component/notification_alert/notification_alert.js

```diff
@@ -0,0 +1,52 @@
+odoo.define('mail_bot.messaging.component.NotificationAlert', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+
+class NotificationAlert extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const isMessagingInitialized = this.env.isMessagingInitialized();
+            return {
+                isMessagingInitialized,
+                isNotificationBlocked: this.isNotificationBlocked,
```

Comment:
More a general comment, but this is going to compute it twice when rendering, storeProps would store the computed value... Probably don't want to change though.

Reply:
We should detect transition messaging not initialized => initialized and changes of window notification permission.